app/ui/src/app/core/providers/integration-support-provider.service.ts

```diff
@@ -183,6 +183,7 @@ export class IntegrationSupportProviderService extends IntegrationSupportService
       .get();
   }
 
+  // @ts-ignore
```

Comment:
Maybe on these can you stick a `TODO` line on?  I think actually we could remove these two functions